test/StakeManager.js

```diff
@@ -392,71 +391,6 @@ describe('StakeManager', function () {
       await assertRevert(tx, 'Nonpositive stake');
     });
 
-    it('Staker should not be able to withdraw after withdraw lock period if voted in withdraw lock period', async function () {
```

Comment:
Hey here can modify this test case, that staker should be able to withdraw even if they have participated in the withdraw lock period.
This will test partial withdraw as well

Reply:
All else looks good

Reply:
Test case has been added